homeassistant/components/zwave.py

```diff
@@ -40,6 +43,12 @@
 SERVICE_TEST_NETWORK = "test_network"
 SERVICE_STOP_NETWORK = "stop_network"
 SERVICE_START_NETWORK = "start_network"
+SERVICE_RENAME_NODE = "rename_node"
+
+RENAME_NODE_SCHEMA = vol.Schema({
+    vol.Required("entityId"): cv.string,
```

Comment:
The key should be `entity_id` (or just `ATTR_ENTITY_ID` which is the same). You can use `cv.entity_id` as validator.